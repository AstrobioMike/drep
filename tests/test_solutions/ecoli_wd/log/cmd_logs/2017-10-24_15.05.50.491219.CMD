nucmer --mum -p /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna -c 65 -g 90 /Users/mattolm/Programs/drep/tests/genomes/Escherichia_coli_Sakai.fna /Users/mattolm/Programs/drep/tests/genomes/Escherichia_coli_Sakai.fna; delta-filter -r -q /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.delta > /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.delta.filtered
