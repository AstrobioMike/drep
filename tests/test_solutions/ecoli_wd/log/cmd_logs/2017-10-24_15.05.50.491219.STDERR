1: PREPARING DATA
2,3: RUNNING mummer AND CREATING CLUSTERS
# reading input file "/Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.ntref" of length 5498451
# construct suffix tree for sequence of length 5498451
# (maximum reference length is 536870908)
# (maximum query length is 4294967295)
# process 54984 characters per dot
#....................................................................................................
# CONSTRUCTIONTIME /Users/mattolm/Scripts/MUMmer3.23/mummer /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.ntref 2.19
# reading input file "/Users/mattolm/Programs/drep/tests/genomes/Escherichia_coli_Sakai.fna" of length 5498450
# matching query-file "/Users/mattolm/Programs/drep/tests/genomes/Escherichia_coli_Sakai.fna"
# against subject-file "/Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.ntref"
# COMPLETETIME /Users/mattolm/Scripts/MUMmer3.23/mummer /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.ntref 6.25
# SPACE /Users/mattolm/Scripts/MUMmer3.23/mummer /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/ANImf_files/Escherichia_coli_Sakai.fna_vs_Escherichia_coli_Sakai.fna.ntref 10.65
4: FINISHING DATA
